executor/builder.go

```diff
@@ -38,13 +38,14 @@ import (
 // executorBuilder builds an Executor from a Plan.
 // The InfoSchema must not change during execution.
 type executorBuilder struct {
-	ctx context.Context
-	is  infoschema.InfoSchema
+	ctx      context.Context
+	is       infoschema.InfoSchema
+	priority int
```

Comment:
Can we put `priority` to `StatementContext`? Then we don't need to put `priority` to multiple structures.